Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -1666,14 +1753,15 @@ LayoutUnit RenderGrid::columnAxisOffsetForChild(const RenderBox& child) const
     gridAreaPositionForChild(child, ForRows, startOfRow, endOfRow);
     LayoutUnit startPosition = startOfRow + marginBeforeForChild(child);
     LayoutUnit columnAxisChildSize = GridLayoutFunctions::isOrthogonalChild(*this, child) ? child.logicalWidth() + child.marginLogicalWidth() : child.logicalHeight() + child.marginLogicalHeight();
+    LayoutUnit masonryOffset = style().gridMasonryRows() ? m_masonryLayout.offsetForChild(child) : 0_lu;
```

Comment:
Use `isMasonryRows(style())` for consistency.

Reply:
Changed